scrape_dalle/config.json

```diff
@@ -2,6 +2,7 @@
     "base_url": "https://discord.com/api/v9",
     "channel_id": "1158354590463447092",
     "limit": 100,
+    "max_chunk_size": 300,
```

Comment:
Is this the same "300" defined as `DATASET_CHUNK_SIZE` above? If yes, let's reuse it maybe?

Reply:
the `DATASET_CHUNK_SIZE` env was my first iteration, but it makes more sense for it to be repo-dependent

Reply:
Thanks for the in-depth review! These comments are very helpful. 
- I'm not using pre-upload anymore since I moved the upload step into the append function, which uses the `HfFileSystem` to upload. If this is recommended I can take a look.
- We store raw images only for some datasets, based on the config. 
 
Here is one with images https://huggingface.co/datasets/laion/dalle-3-dataset. I still need get the readme updated as well for the dataset viewer here